content/pages/features.html

```diff
@@ -5,152 +5,157 @@
-                            Plan. These are the actual scenarios that will be tested.
-                        </li>
-
-                        <li>
-                            <i class="fa fa-check"></i>
-                            <strong>Execute:</strong> create Test Run for each build and environment
-                            that will be tested to record the results.
-                        </li>
-                    </ul>
-                </div>
 
-                <div style="float: right; clear:right;margin-bottom: 5rem;">
-                    <img src="/images/features/testplan.png" class="margin0 shadow">
+                <div class="col-2"></div>
+            </section>
+
+            <!-- Test Plan -->
+            <section class="col-12 stripe-gray">
+                <div class="col-12 screenshot-title">
+                    <h3>A Test Plan is a collection of Test Cases</h3>
                 </div>
-            </div>
-
-            <div class="col-2"></div>
-        </section>
-
-        <!-- Test Run -->
-        <section class="col-12 secondary">
-            <div class="col-2"></div>
-
-            <div class="col-8" style="padding-top: 2rem;">
-                <div class="w50" style="float: left;">
-                    <div class="about-panel-details">
-                        Execute Test Cases and collect their results!
-                        Tagging and Environment properties help you identify
-                        different execution environments. Use the Test Run
-                        status widget to monitor progress and generate a report!
+
+                <div class="col-2"></div>
+
+                <div class="col-8">
+                    <div style="float:left" class="col-3 w25">
+                        <ul class="check-list">
+                            <li>
+                                <i class="fa fa-check"></i>
+                                <strong>Document:</strong> use IEEE 829 format or simply describe what
+                                you are planning to do and what your strategy is.
+                            </li>
+
+                            <li>
+                                <i class="fa fa-check"></i>
+                                <strong>Test Cases:</strong> review and add Test Cases to the
+                                Plan. These are the actual scenarios that will be tested.
+                            </li>
+
+                            <li>
+                                <i class="fa fa-check"></i>
+                                <strong>Execute:</strong> create Test Run for each build and environment
+                                that will be tested to record the results.
+                            </li>
+                        </ul>
                     </div>
+                    
+                    <div class="col-2"></div>
+
+                    <div style="margin-bottom: 5rem;" class="col-3">
```

Comment:
Compare how the current layout looks vs. what's on the live website and notice that the image goes a bit to the right on a regular desktop screen. This is because of the style nesting you have introduced. The col-3 class here means 25% of the encapsulating col-8 above (which is 66,66%). This is not 25% of the overall page width!

I think you should work with the previous styles, but to make things easier to spot you will have to apply the body/features change first so the diff is easier to review.